tests/integration/test_cdn_renewal.py

```diff
@@ -10,9 +10,10 @@
     CdnCertificate,
     CdnChallenge,
 )
-from renewer.tasks import cdn, iam, letsencrypt, s3
+from renewer.tasks import cdn, iam, letsencrypt, s3, renewals
 
 from tests.lib.fake_iam import FakeIAM
+from tests.lib.cdn_fixtures import make_route, make_cert
```

Comment:
Do you need to import the `alb_fixtures` versions, too?  Or are these all just `CdnRoute`-based tests in here?

Reply:
this is the cdn renewal file, so we shouldn't have anything referencing albs here

Reply:
D'oh, you're right, didn't look closely at the file name.